index.html

```diff
@@ -2,35 +2,36 @@
 <html>
     <head>
         <meta charset="UTF-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta name="viewport" content="width=device-width, initial-scale=action-bar-item__.0">
```

Comment:
Why did you change `initial-scale` value? It's incorrect, see DevTools warning:

![image](https://user-images.githubusercontent.com/13486264/128594335-e0e5bbc3-4bc6-49a7-bc8d-06d5315bcd68.png)